registration/TOS.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+  <title>Terms Of Service</title>
+</head>
+<body>
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque in tincidunt nisl, id mollis leo. Vestibulum rhoncus purus ac mollis ultricies. Curabitur iaculis eu lacus et sodales. Morbi ornare, ipsum sed eleifend placerat, orci nibh venenatis est, et eleifend turpis dui ac enim. Pellentesque dapibus ante erat, vel pulvinar dolor rhoncus vitae. Pellentesque vel eros tortor. Duis ut porta dolor. Fusce vel mi in ex dapibus vulputate. Donec purus orci, sagittis vel mi id, mollis efficitur lacus.
+
+Aliquam erat volutpat. Etiam eros neque, blandit vel erat eu, commodo ultrices diam. Sed non finibus sem. Curabitur sagittis dui augue, a pellentesque leo sagittis vitae. Nullam nec varius dolor, vel ullamcorper turpis. Nullam interdum ornare tellus, eu luctus arcu tincidunt at. Mauris lacinia magna sed dolor ultricies, quis posuere odio malesuada. Donec sit amet cursus sapien. Praesent euismod felis quis diam gravida, varius tempus augue placerat. Quisque porttitor, dui ac facilisis pellentesque, dolor orci bibendum nisl, nec convallis lacus nibh ut felis.
+
+Maecenas lacus enim, sagittis sit amet massa id, maximus condimentum eros. Nulla facilisi. Duis facilisis justo lacus, in vestibulum ipsum blandit eu. Vestibulum sagittis tristique diam in eleifend. Ut malesuada ante nec turpis elementum facilisis. Pellentesque hendrerit et augue id suscipit. Proin iaculis elementum enim in lacinia. Phasellus at placerat felis. Integer a justo dapibus, interdum massa at, facilisis justo. Aliquam non vestibulum magna. Proin a orci vitae enim volutpat sagittis. Ut mattis enim non ornare efficitur. In vitae metus ac tellus fermentum tempor. Mauris sed efficitur augue. Curabitur eu semper dolor, ut finibus mauris.
+
+Nunc lacus dolor, lobortis a convallis et, rhoncus a nisl. Donec ac augue imperdiet, venenatis diam eu, consequat risus. Phasellus condimentum consequat tellus, quis hendrerit ex pulvinar eget. Pellentesque malesuada vel ligula id aliquam. Cras diam nulla, viverra vitae est vitae, porta tincidunt neque. Integer a porttitor nibh. Vestibulum vitae facilisis eros. Etiam fermentum convallis dolor, non commodo ante pretium id. Nam dapibus sed eros et pulvinar. Nulla eget purus rhoncus, malesuada ligula vel, posuere nulla. Praesent non sem sed libero sodales pellentesque eu ac erat. Integer in metus velit.
+
+Integer eget sapien mauris. Donec posuere ut libero sed sodales. Nunc maximus sed lorem id egestas. Sed lacus quam, finibus ac vestibulum eu, interdum id massa. Morbi eget eros metus. Sed et semper diam. Mauris eu odio nec enim tempor pellentesque. Curabitur quis blandit arcu. Praesent in eros metus. 
```

Comment:
I think we should leave the body empty for now instead of putting in random text? What do you think?